### PR TITLE
FORMS-235 text changes

### PR DESCRIPTION
**Pull request by kakanisridhar**
# Description

- Create component landing page changed to include help text below links

## Type of change
- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
- [X] Unit tests

# Checklist:

- [X] My changes do not introduce any new linting errors
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation and versioning
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I have rebased onto master and there are no code conflicts
- [ ] I have checked deployments are working in all environments
- [ ] I have updated the architecture diagrams as per Contribute.md

### CODE REVIEW

File: designer/client/components/ComponentCreate/ComponentCreateList.tsx [hunk 1]
```diff
@@ -59,13 +57,17 @@ export const ComponentCreateList = ({ onSelectComponent }: Props) => {
                 >
                   {i18n(`fieldTypeToName.${component.type}`)}
                 </a>
+                <p className="govuk-body-s">
```
Comment (bertrandmc) [inline designer/client/components/ComponentCreate/ComponentCreateList.tsx:60]: I think we should be using `<div className="govuk-hint">{i18n(...)}</div>` for the hints, so they follow GOV UK style guide

**Review by bertrandmc — CHANGES_REQUESTED** [review]

File: designer/client/components/ComponentCreate/__tests__/ComponentCreateList.jest.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import { initI18n } from "../../../i18n";
+import { ComponentCreateList } from "../ComponentCreateList";
+
+initI18n();
+
+describe("ComponentCreateList", () => {
```
Comment (bertrandmc) [inline designer/client/components/ComponentCreate/__tests__/ComponentCreateList.jest.tsx:8]: I think we had agreed snapshots were out and instead we'd be using `expect(getByText("Delete list")).toBeInTheDocument();`? Could you please confirm with Enda?

**Review by bertrandmc — APPROVED** [review]
Just a question about snapshots, happy to approve if Enda confirms snapshots are allowed.

File: designer/client/components/ComponentCreate/__tests__/ComponentCreateList.jest.tsx (hunk 2 above)
Comment (kakanisridhar) [reply designer/client/components/ComponentCreate/__tests__/ComponentCreateList.jest.tsx:8]: i think there snapshot is better for this as there are 25 long texts to be checked if getByText used

**Review by bertrandmc — APPROVED** [review]

**Pull request merged**